docs/09_firebase_deployment.md

```diff
@@ -0,0 +1,201 @@
+3. Choose the app nickname and check `Also set up Firebase Hosting for this app`.
+4. Select from the dropdown menu, or create a new site for your application and tap the `Register app` to proceed.
+5. Skip the `Configure Firebase SDK` step. We will return to Firebase SDK configuration a bit later in [Firebase SDK configuration](#Firebase-SKD-configuration).
+6. Tap on the `Next` button and follow instructions to install the Firebase CLI.
+7. Skip the `Deploy to Firebase Hosting` and tap on the `Continue to console` to finish configuring your Firebase Web application.
+ The deployment process described more detailed in
+  [Building and deploying the application to the Firebase Hosting](#Building-and-deploying-the-application-to-the-Firebase-Hosting) section.
+ 
+Finally, your Firebase project configured ant it's time to configure the Firebase SDK in your Flutter for Web application.
+
+## Firebase SKD configuration
+
+To configure the Flutter for Web application to use recently created Firestore Database follow the next steps: 
+
+1. Open the [Firebase console](https://console.firebase.google.com/), choose your project 
+and go to the project setting (tap on the setting gear icon near the `Project Overview` on top of the left panel and select `Project settings`.
+2. Scroll down and find your Firebase Web Application. 
+3. Go to `Firebase SDK snippet` of your application, select `Config` and copy the generated code.
+4. Go to the `web/index.js` file in the application directory and replace this piece of code with the copied one in step 3: 
+    ```
+        var firebaseConfig = {
+          apiKey: "AIzaSyCkM-7WEAb9GGCjKQNChi5MD2pqrcRanzo",
+          authDomain: "metrics-d9c67.firebaseapp.com",
+          databaseURL: "https://metrics-d9c67.firebaseio.com",
+          projectId: "metrics-d9c67",
+          storageBucket: "metrics-d9c67.appspot.com",
+          messagingSenderId: "650500796855",
+          appId: "1:650500796855:web:65a4615a28f3d88e8bb832",
+          measurementId: "G-3DB4JFLKHQ"
+        };
+    ```
+
+Finally, you have a configured Flutter application that works with your Firebase instance.
+It's time to deploy your Flutter application to the Firebase Hosting!
+ 
+## Building and deploying the application to the Firebase Hosting
+
+### Preparing your environment 
+Before deploying metrics application, make sure you have the correct Flutter version installed,
+ by running the `flutter --version` command. You should have `v1.15.3` installed. 
+
+To switch the Flutter to the required version, you should run the `flutter version 1.15.3` command.
+
+Also, you should enable flutter web support by running the `flutter config --enable-web` command.
+
+### Building and deploying Flutter application
+ 
+1. Open the terminal and navigate to the metrics project folder.
+2. Run `firebase login` command and follow the instructions to log in to the Firebase CLI with your Google account.
```

Comment:
Should we install `firebase` first?

Reply:
We have installed the Firebase CLI earlier - during creating the Firebase Web project (firebase configuration section, step 6), so it should already be installed on this step.

Reply:
Thanks!